vite-plugin/src/index.ts

```diff
@@ -22,32 +22,46 @@ function buildSuccessPlugin(callback: () => void | Promise<void>): Plugin {
 
 export default function jahia(
   options: {
+    /**
+     * Source directory where to find all source files.
+     *
+     * @default "src"
+     */
+    inputDir?: string;
+
+    /**
+     * Directory where to put all built files.
+     *
+     * @default "dist"
+     */
+    outputDir?: string;
+
+    /**
+     * Directory where to put all built assets (e.g. images, fonts, etc.).
+     *
+     * This directory will be nested in the `outputDir` directory.
```

Comment:
isn't there a risk of conflicts in case the source folder contains an `assets` folder? 🤔